packages/fxa-auth-server/scripts/write-emails-to-disk.js

```diff
@@ -110,6 +110,8 @@ function sendMail(mailer, messageToSend) {
     productName: 'Firefox Fortress',
     planEmailIconURL: 'http://placekitten.com/512/512',
     planDownloadURL: 'http://getfirefox.com/',
+    planInterval: 'week',
```

Comment:
@peiying2 

Re: https://pontoon.mozilla.org/it/firefox-accounts/all-resources/?status=missing&string=224251

This looks hard-coded to me. Besides that, this structure ("$num $period subscription") won't work in any language but English.